examples/amp-story/interactives.html

```diff
@@ -429,7 +429,7 @@ <h1 style="color:white;text-align:center;line-height:40px;margin-bottom:10px;">W
               theme="light"
               prompt-text="What is your favorite food?"
               option-1-text="Tacos"
-              option-2-text="Bacon"
+              option-2-text="Bacon is my favourite food in the world and I eat it everyday"
```

Comment:
🥓 🥓 🥓 🥓 🥓 🥓 🥓 🥓 🥓 🥓 🥓

Reply:
Purely done in order to reach 2 lines on a poll, nothing to look over here 👀